apidoc/Titanium/UI/Label.yml

```diff
@@ -166,8 +166,13 @@ properties:
     summary: Minimum font size when the font is sized based on the contents. 
     description: |
         Enables font scaling to fit and forces the label content to be limited to a single line.
+        
+        On Android, setting this property will force the `ellipsize` property to be
```

Comment:
keeping in format with existing docs:
+ make `ellipsize` into link to property link?
+ either edit into **or** append `default: <Titanium.UI.TEXT_ELLIPSIZE_TRUNCATE_END> on Android`...and so and so behavior on iOS, etc...